src/components/ProgramRecord/ProgramRecord.scss

```diff
@@ -44,20 +49,22 @@
             }
 
             .program-status {
-                @extend .d-flex, .align-items-center;
+                display: flex !important;
+                align-items: center !important;
                 gap: 0.5rem;
 
                 span.badge {
-                    @extend .d-flex;
+                    display: flex !important;
                 }
 
                 span.updated {
-                    @extend .small;
+                    font-size: var(--pgn-typography-font-size-sm);
+                    font-weight: var(--pgn-typography-font-weight-normal);
                 }
             }
 
             .learner-info {
-                @extend .d-flex;
+                display: flex !important;
```

Comment:
Based on https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 moving away from `@extend` in this way is good as-is!